### ISSUE TITLE
Code blocks written in a .md file isn't rendered as one, in the browser

### ISSUE THREAD

**Issue by Jarmos-san** [getpelican/pelican#2832]
- [X] I have read the [Filing Issues](https://docs.getpelican.com/en/latest/contribute.html#filing-issues) and subsequent “How to Get Help” sections of the documentation.
- [X] I have searched the [issues](https://github.com/getpelican/pelican/issues?q=is%3Aissue) (including closed ones) and believe that this is not a duplicate.

- **OS version and name**: Windows 10
- **Python version**: 3.8.5
- **Pelican version**: 4.5.3
- **Link to theme**: https://github.com/jody-frankowski/blue-penguin
- **Links to plugins**: 
- **Link to your site**: 
- **Link to your source**: https://github.com/Jarmos-san/blog (haven't pushed the latest dev build, yet)
- **Link to a [Gist](https://gist.github.com/) with the contents of your settings file**: https://gist.github.com/Jarmos-san/7dcb7f00ba2b56cdc38c03f44aa0565d

## Issue

Code blocks doesn't render as it should in the browser. Screenshot attached for reference.

![image](https://user-images.githubusercontent.com/31373860/103371311-2568cc00-4af5-11eb-86bf-ab0e14012ca4.png)

As [suggested in the docs](https://docs.getpelican.com/en/latest/content.html#syntax-highlighting), using a path-less shebang with the identifier should've done the trick.

Expected output something like this (in addition to the line numbers):

```python
import os
import sys

print('some text here')
```

Each statement should be on it's own line but it isn't so in the browser. Besides, I'm also using Pelican with Markdown.

I was wondering, if this is a bug specific to Python-Markdown or something else. I didn't have this issue while using reStructuredText files. So I'm facing this problem only after I started migrating all my articles to Markdown.

**Comment by avaris** [comment]
It's your theme, and how you style `code` elements. Specifically [this line](https://github.com/Jarmos-san/blog/blob/950ef6da74ec9d2402774944f50725b0ea661048/themes/blue-penguin/static/css/screen.css#L326). I also noticed that the theme in your repo is not same as the theme you linked in the issue. Possibly, the original doesn't have this issue.

**Comment by Jarmos-san** [comment]
>It's your theme, and how you style `code` elements.

Yes, you're right. I cloned a theme from [Pelican Themes](http://www.pelicanthemes.com/) & the author wasn't responsive about some other bugs I came across with the theme. So I had to "copy" it to my site instead & fix as many bugs as I possibly could. I'm closing the issue as it doesn't in any way appear to be related to Pelican.

Regardless, here's what I observed. Based on [this SO answer](https://stackoverflow.com/a/63333246/8604951), setting `code { white-space: pre }` does _fix the issue_. But, the browser somehow resets to `white-space: nowrap`. I'm no CSS mastermind, so I've no clue what's going on here. Hence, I'm sticking to embedding Gists instead for now.

@avaris thanks for validating my suspicions.
